pkg/debug/scripts/collect-info.sh

```diff
@@ -234,6 +234,11 @@ collect_pillar_backtraces()
 
     kill $pid
 
+    # collect memory backtrace
+    eve http-debug > /dev/null 2>&1
+    eve exec debug curl "http://127.1:6543/debug/pprof/heap?debug=1" > "$DIR/pillar-memory-backtraces"
```

Comment:
Are all these synchronous commands? I mean don't we have to wait until the http server is started?

Reply:
I moved `eve http-debug` upwards so now it should have enough time to open up the port